addons/mail/static/tests/discuss_app/sidebar_tests.js

```diff
@@ -1158,3 +1158,19 @@ QUnit.test("Unpinning channel closes its chat window", async () => {
     await openFormView("discuss.channel");
     await contains(".o-mail-ChatWindow", { count: 0, text: "Sales" });
 });
+
+QUnit.test("Update channel data via bus notification", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "Sales" });
+    const { openDiscuss } = await start();
+    openDiscuss();
+    await contains(".o-mail-DiscussSidebarChannel", { text: "Sales" });
+    pyEnv["bus.bus"]._sendone(pyEnv.currentPartner, "mail.record/insert", {
+        Thread: {
+            id: channelId,
+            model: "discuss.channel",
+            name: "test",
+        },
+    });
+    await contains(".o-mail-DiscussSidebarChannel", { text: "test" });
+});
```

Comment:
You could make a multi-tab test, and have one tab change the name, and see the name is changed on the other tab.
The test would need to adapt mock server accordingly, as you changed the code in python this makes sense